crypto/rsa/rsa_oaep.c

```diff
@@ -129,6 +121,15 @@ int ossl_rsa_padding_add_PKCS1_OAEP_mgf1_ex(unsigned char *to, int tlen,
     return rv;
 }
 
+int RSA_padding_add_PKCS1_OAEP(unsigned char *to, int tlen,
+                               const unsigned char *from, int flen,
+                               const unsigned char *param, int plen)
+{
+    return ossl_rsa_padding_add_PKCS1_OAEP_mgf1_ex(to, tlen, from, flen,
+                                                   param, plen, NULL, NULL);
+}
```

Comment:
Why are we moving `RSA_padding_add_PKCS1_OAEP` down in this source file? I see we're moving it to after the function it calls, but I guess this somehow compiled before anyway, perhaps there's a prototype somewhere?

Reply:
Yes, this is a warning removal. Without these there are so many warnings it become hard to see real bugs.